core/modules/map.rs

```diff
@@ -901,7 +925,7 @@ impl ModuleMap {
     let load = RecursiveModuleLoad::dynamic_import(
       specifier,
       referrer,
-      asserted_module_type,
+      asserted_module_type.clone(),
```

Comment:
All of these clones will be cheap, except if using a non-standard assertion type.

Reply:
What do you mean by non-standard assertion type?

Reply:
Anything other than JSON or JavaScriptOrWasm. Those two (the only ones we actually use right now) will be cheap clones.